appveyor.yml

```diff
@@ -4,6 +4,7 @@ image: Visual Studio 2015
 
 cache:
   - '%LocalAppData%\Microsoft\dotnet'
+  - '%HOMEDRIVE%%HOMEPATH%\.nuget\packages'
```

Comment:
Out of curiosity, what is the cache invalidation and update story for this? NuGet packages periodically get updated so I just want to make sure that it will correctly handle updates and not continue using a stale package cache.